conduit/scripts/entrypoint.sh

```diff
@@ -37,6 +37,7 @@ export \
   AIRFLOW__CORE__LOAD_EXAMPLES \
   AIRFLOW__CORE__SQL_ALCHEMY_CONN \
 
+export AIRFLOW_HOME=/root
```

Comment:
@dxenes1

Reply:
The recent versions of Airflow seem to have stepped away from defining the home directory in the CFG file to prevent conflicts with custom CFG files. (such as in our case) Reference:

https://issues.apache.org/jira/browse/AIRFLOW-3743

Exposing the home directory in the entrypoint allows airflow to find the right CFG file and prevents the issue that Luis and I had with the webserver unable to find the DAGs.